plugins/coverbrowser.koplugin/main.lua

```diff
@@ -155,6 +155,23 @@ function CoverBrowser:addToMainMenu(menu_items)
     -- next to Classic mode settings
     if menu_items.filebrowser_settings == nil then return end
     local fc = self.ui.file_chooser
+    local function genSeriesSubMenuItem(text_, value_)
+        return {
+            text = text_,
```

Comment:
Do we need to use `text_` and `value_` (with a trailing underscore) ?
(It surprised me, so I don't think we anywhere use such naming, even to avoid conflict with upper scoped variables if there really were some.)

Reply:
Several times before, I encountered conflicts in naming the standard args (like text, file etc) in local function.
So I prefer to not think about it and just give unique names.

Reply:
https://github.com/koreader/koreader/blob/6e84503b3dbfcd5b249065e3c3b038ae3b7428ac/frontend/apps/reader/modules/readerhighlight.lua#L2064

Reply:
Well, I'd rather like you to think a bit and see such naming is not needed in such an obvious context :) and so we don't start a new trend and future developers think this is the thing to do :)

Reply:
Good unique names, I believe.